### ISSUE TITLE
incorrect quiz answer

### ISSUE THREAD

**Issue by xpcai2010** [udacity/sdc-issue-reports#1392]
Dear Instructor:
Self-Driving Car Nano Degree - Lesson 10: Tensor Flow, the answer to the Quiz: TensorFlow Softmax Workspaces.
For question 2 of 2: the course gives the correct answer as first one. But I think the second answer should be the right one.
